app/metrics.yaml

```diff
@@ -132,3 +133,112 @@ tabs:
       - fxr-telemetry@mozilla.com
       - manmartin@mozilla.com
     expires: "2020-05-01"
+
+fxa:
+  sign_in:
+    type: counter
+    description: >
+      Number of users who sign in
+    send_in_pings:
+      - baseline
```

Comment:
It is not clear if we should send our data in pings or the end of session. I suppose it should be in pings.

Reply:
I've migrated most of the FxA metrics to events, it's how Fenix is doing it: https://github.com/mozilla-mobile/fenix/pull/2745#issuecomment-494918532

This are sent through pings so I've removed the end of session.